auth/auth.go

```diff
@@ -41,9 +41,28 @@ const (
 	// so we give it 15 seconds to refresh it's cache before attempting to refresh a token.
 	defaultExpiryDelta = 225 * time.Second
 
+	// nonBlockingRefreshTimeout is the timeout for asynchronous refresh of a
+	// token.
+	nonBlockingRefreshTimeout = 30 * time.Second
```

Comment:
it looks like this was forgotten to be used.

Reply:
Removed, thanks!